modules/bitgo/test/v2/unit/coins/abstractUtxoCoin.ts

```diff
@@ -583,4 +583,25 @@ describe('Abstract UTXO Coin:', () => {
       bitcoinMock.restore();
     });
   });
+
+  describe('Verify paygo output when explaining psbt transaction', function () {
+    const bitgo: BitGo = TestBitGo.decorate(BitGo, { env: 'mock' });
+    let coin: AbstractUtxoCoin;
+
+    beforeEach(() => {
+      coin = bitgo.coin('tbtc4') as AbstractUtxoCoin;
+    });
+
+    it('should detect and verify paygo address proof in PSBT', async function () {
+      // Mock parameters for explainTransaction
+      const psbtTxHexString =
```

Comment:
put this sort of data into json files

explain where it comes from and how how to create this data